src/core/utils/assets.ts

```diff
@@ -125,10 +125,12 @@ export function parseAsset({
     colors: asset.colors,
     standard,
     networks: asset.networks,
-    bridging: {
-      isBridgeable: asset.bridging.bridgeable,
-      networks: asset.bridging.networks,
-    },
+    ...('bridging' in asset && {
+      bridging: {
+        isBridgeable: !!asset?.bridging?.bridgeable,
+        networks: asset?.bridging?.networks || undefined,
```

Comment:
after the `'bridging' in asset` check these others are not needed anymore
```suggestion
        isBridgeable: !!asset.bridging.bridgeable,
        networks: asset.bridging.networks,
```